packages/sdk/src/reads/getIdleGovTokensAmounts.ts

```diff
@@ -0,0 +1,63 @@
+import { type ReadContractParameters, readContractParameters } from "../utils/viem.js";
+import type { Address, PublicClient } from "viem";
+
+const abi = [
+  {
+    constant: true,
+    inputs: [{ internalType: "address", name: "_usr", type: "address" }],
```

Comment:
should this be `name: "_user"`?

Reply:
not a typo, abi is copied from etherscan